### ISSUE TITLE
enable all launch options at once

### ISSUE THREAD

**Issue by Martinligabue** [dylanaraps/neofetch#2126]
## Description

I was wondering if it would be helpful a launch option to enable every possible output, without having to enable each one manually, like disk, temperatures, battery, etc...
and maybe also one to disable every thing so just specify which to enable

**Comment by evrial** [comment]
Yes, please. Who fucking cares about stupid fonts and themes but Disk and Battery and local/public IP is disabled.
Nerds, just make SANE configs out of the box.
